### DISCUSSION TITLE
C128 speed on C64 mode?

### DISCUSSION THREAD

**Discussion by adaumann** [drmortalwombat/oscar64#165] [General]
I have a question regarding to C128. In some games the faster CPU of the C128 could be used to enhance the speed of C64, I have not found any useful documentation. Is this possible or an idea to integrate in Oscar64?

**Comment by theflyingape** [comment]
You speak of the speed hack that switches C128 CPU into 2 mHz mode _while_ the VIC-II vertical scan line is blanking the border color... the result is ~20% overall performance gain. Some games will blank VIC-II entirely during a "thinking" phase to enjoy the 2x speed for its crunching.
So this really depends on your needs, not the compiler. I suggest you engage a user forum like Lemon64 and narrow in on what you'd like to implement. There are many examples to accomplish what you are looking for. Enjoy the journey. :+1:

**Comment by drmortalwombat** [comment]
This might be a good idea for the sample set

**Comment by drmortalwombat** [comment]
https://github.com/drmortalwombat/OscarTutorials/tree/main/1425%20RasterIRQTurbo

enjoy

**Reply by adaumann** [reply]
Great, I will try to integrate
